.github/workflows/consuming.yml

```diff
@@ -102,8 +102,47 @@ jobs:
       - name: Make sure ${{ matrix.project }} is using the built Shipyard image
         run: sed -i 's/shipyard-dapper-base:*.*/shipyard-dapper-base:dev/' ${{ matrix.project }}/Dockerfile.dapper
 
-      - name: Run all linters
-        run: make -C ${{ matrix.project }} lint
+      - name: Run gitlint
+        if: always()
+        run: make -C ${{ matrix.project }} gitlint
+
+      - name: Run golangci-lint
+        if: ${{ always() && matrix.project != 'submariner-charts' }}
+        run: make -C ${{ matrix.project }} golangci-lint
+
+      - name: Run markdownlint
+        if: always()
+        run: make -C ${{ matrix.project }} markdownlint
+
+      - name: Run packagedoc-lint
+        if: >-
+          ${{
+          always()
+          && matrix.project != 'cloud-prepare'
+          && matrix.project != 'coastguard'
+          && matrix.project != 'lighthouse'
+          && matrix.project != 'submariner-charts'
+          && matrix.project != 'submariner-operator'
```

Comment:
`packagedoc-lint` has no effect on `submariner-charts`; the rest should be fixed.